onnxruntime/test/providers/cpu/tensor/scatter_op_test.cc

```diff
@@ -174,30 +167,14 @@ static void scatter_indices_updates_dont_match(const char* op_name, int op_versi
   test.AddInput<int64_t>("indices", {1, 3}, {1, 3, 3});
   test.AddInput<float>("updates", {1, 2}, {1.1f, 2.1f});
   test.AddOutput<float>("y", {1, 5}, {1.0f, 1.1f, 3.0f, 2.1f, 5.0f});
-  test.Run(OpTester::ExpectResult::kExpectFailure, "Indices vs updates dimensions differs at position=1 3 vs 2", {kTensorrtExecutionProvider});
+  test.Run(OpTester::ExpectResult::kExpectFailure, "", {kTensorrtExecutionProvider});
 }
 
 TEST(Scatter, IndicesUpdatesDontMatch) {
   scatter_indices_updates_dont_match("Scatter", 9);
   scatter_indices_updates_dont_match("ScatterElements", 11);
 }
 
```

Comment:
"TEST(Scatter, ValidAxis) " and other tests are removed by intention?

Reply:
All valid cases are moved to RunTestWrapper(), which includes all old test cases (same shapes and axis) and lot's of new cases I added to cover different coalesce cases. Only those invalid cases that produce error messages are kept.